x-pack/auditbeat/module/system/host/host.go

```diff
@@ -315,11 +323,59 @@ func hostEvent(host *Host, eventType string, action eventAction) mb.Event {
 				"type":   eventType,
 				"action": action.String(),
 			},
+			"message": hostMessage(host, action),
 		},
 		MetricSetFields: host.toMapStr(),
 	}
 }
 
+func hostMessage(host *Host, action eventAction) string {
+	// Hostname + IP of the first non-loopback interface.
+	hostString := fmt.Sprintf("%v (IP: %v)", host.info.Hostname, ipString(host.addrs[0]))
+
+	var message string
+	switch action {
+	case eventActionHost:
+		message = fmt.Sprintf("%v host %v is up for %v",
+			host.info.OS.Name, hostString, fmtDuration(host.uptime))
+	case eventActionIDChanged:
+		message = fmt.Sprintf("ID of host %v has changed", hostString)
+	case eventActionReboot:
+		message = fmt.Sprintf("Host %v restarted", hostString)
+	case eventActionHostnameChanged:
+		message = fmt.Sprintf("Hostname changed to %v", hostString)
+	case eventActionHostChanged:
+		message = fmt.Sprintf("Host %v changed", hostString)
+	}
+
+	return message
+}
+
+func fmtDuration(d time.Duration) string {
```

Comment:
Functions like this should get some unit tests. Fine with adding them later in this case, but normally we'd expect them with the PR.